QUICKSTART.rst

```diff
@@ -3,120 +3,153 @@
+   b. Set up your environment for Chapel's Quick Start mode.
+      If you are using a shell other than bash,
+      see quickstart-with-other-shells_ below.
 
-==================================== ==========================================
-**If you use:**                       **then type:**
------------------------------------- ------------------------------------------
-a bash-compatible shell (bash/zsh)   ``source util/quickstart/setchplenv.bash``
-a csh-compatible shell (csh/tcsh)    ``source util/quickstart/setchplenv.csh``
-the fish shell (fish)                ``. util/quickstart/setchplenv.fish``
-the Bourne shell (sh)                ``. util/quickstart/setchplenv.sh``
-==================================== ==========================================
+      .. code-block:: bash
 
-   Note that there is no requirement to use these scripts long-term,
-   they are merely designed to help new users get their environment
-   variables and paths set up quickly.  Long-term, users may want to
-   copy and paste these settings into their dotfiles or set them in
-   other ways.  See `doc/chplenv.rst`_ for a complete description of
-   Chapel's environment variables and their expected values.
+         source util/quickstart/setchplenv.bash
 
-.. _doc/chplenv.rst: http://chapel.cray.com/docs/1.13/usingchapel/chplenv.html
+   c. Use GNU make to build Chapel.
+      On some systems, you will have to use gmake.
+      See `doc/building.rst`_ for more information about building Chapel.
 
+      .. code-block:: bash
 
-3) Build the compiler and runtime libraries using:
+         make
 
-        ``gmake``
+   d. Optionally, check that your Chapel build is working correctly
 
-   or if your default make is GNU make compatible (as on Mac OS X or
-   Cygwin), use:
+      .. code-block:: bash
 
-        ``make``
+         make check
 
-   Parallel builds (e.g. ``gmake -j``) are also supported.
 
+3) Compile an example program:
 
-4) ``csh``/``tcsh`` users only: Update your shell's path hash table using:
+   .. code-block:: bash
 
-        ``rehash``
+      chpl -o hello examples/hello.chpl
 
+4) Run the resulting executable:
 
-5) Optionally, check that your Chapel installation is working correctly:
+   .. code-block:: bash
 
-        ``gmake check``
+      ./hello
 
-   or:
+5) Experiment with Chapel in this Quickstart mode to your heart's
+   content.  Once you are comfortable with Chapel and interested in
+   using a full-featured version in the preferred configuration, see
+   the next section.
 
-        ``make check``
 
+.. _using-a-more-full-featured-chapel:
 
-6) Compile an example program:
+Using a More Full-Featured Chapel
+---------------------------------
 
-        ``chpl -o hello examples/hello.chpl``
+To use Chapel in a more full-featured and preferred configuration,
+you will need to rebuild Chapel from source in a different configuration.
 
+a) Open up a new shell to avoid inheriting the previous environment
+   settings.
```

Comment:
The use of (a) and (b) seem odd to me here since they're top-level actions, but I'm guessing it was to avoid confusion with the reference to steps 2-5.  What about just bullets since there are just two of them?